tasks/create_ordered_kafka_groups.yml

```
@@ -19,13 +19,13 @@
     owner: "{{kafka_broker_user}}"
     group: "{{kafka_broker_group}}"
   when:
-    - "'zookeeper.ssl.client.enable=true' in slurped_properties.content|b64decode"
+    - "'zookeeper.ssl.client.enable = true' in slurped_properties.content|b64decode"
```

```suggestion
    - "'zookeeper.ssl.client.enable = true' in slurped_properties.content|b64decode or 'zookeeper.ssl.client.enable = true' in slurped_properties.content|b64decode"
```

i think we need an or condition. do you have secrets protection enabled?